example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.40'
+    ext.kotlin_version = '1.6.0'
```

Comment:
Is this for the upgrade?

Reply:
Have the same issue when building the android app like this https://github.com/flutter/flutter/issues/83834 
I think the issue is caused by I updated my android studio recently and the current kotlin plugin version is 1.6.0 which is not compatible with kotlin version (1.3.40) in the project. 
It shouldn't be an issue for other customers.